node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java

```diff
@@ -82,6 +82,20 @@ private NodeResources specifiedOrDefaultNodeResources(Optional<NodeResources> re
         }
     }
 
+    private NodeResources defaultNodeResources(ClusterSpec.Type clusterType) {
```

Comment:
This method reproduces the capacity policies previously encoded in the zone enum. I'm relieved to discover that it's perfectly uniform for main, but the rest seems a little odd/ad hoc. 

Can you suggest improvements?